servicetalk-dependencies/build.gradle

```diff
@@ -96,6 +93,16 @@ publishing {
           system = 'ServiceTalk CI'
           url = "${ciManagementUrl}"
         }
+        // Gradle ignores dependencies constraints with classifiers when it generates pom.xml. To workaround it, we are
+        // forced to use withXml. See https://github.com/gradle/gradle/issues/8561
+        withXml {
+          Node pomNode = asNode()
+          Node dependencies = pomNode.dependencyManagement.dependencies.get(0)
```

Comment:
Is there any possibility that this node won't already exist and needs to be created?

Reply:
No, `withXml` part is invoked after the pom.xml is generated. It always has some `dependencies`.